oat-sa/extension-tao-itemqti
# Fix/adf-1766/defined-item-width

related to https://oat-sa.atlassian.net/browse/ADF-1766

### Description

If Item styles set to use defined width for an item, the value gets updated after item is opened for authoring again.
This PR fixes the issue setting the item

### How to test

Follow the steps :

- Create an item and open it for authoring
- Open Item styles and choose radio button of the Item width to be Defined width 
- Set the width to , i.e. 1200
- Save the item
- Open item for editing again
- Open Item Styles, check the width - it will be reduced

Ensure the style is saved correctly
